spec/controllers/prison/visits_controller_spec.rb

```diff
@@ -71,6 +71,42 @@
 
         it { is_expected.to redirect_to(prison_inbox_path) }
       end
+
+      context 'when there is an api error when booking to nomis' do
```

Comment:
there need to be an outer scope called "When booking to nomis" as it seems to be repeated in the next context block, that would share the setup of `let(:stadd_response)` and `let(:booking_response)`